integration-libs/cdc/components/public_api.ts

```diff
@@ -1,4 +1,5 @@
 /*
  * Public API Surface for CDC
  */
-export * from './index';
+export * from './cdc-components.module';
+export * from './gigya-raas/index';
```

Comment:
Should index.ts be deleted like elsewhere?

Reply:
should deleted